plugin/config_test.go

```diff
@@ -101,6 +101,149 @@ output: stdout
 	require.Equal(t, "noop", noop.OperatorType)
 }
 
+type PluginIDTestCase struct {
```

Comment:
Aside from changes in this file, are the other changes necessary to make these tests pass? If so, what functionality do they change?